inc/mf2.php

```diff
@@ -34,6 +34,33 @@ function independent_publisher_mf2_post_class( $classes ) {
 
 add_filter( 'post_class', 'independent_publisher_mf2_post_class' );
 
+/**  
+ * Wraps the_content in e-content
+ *
+ */
+function independent_publisher_the_content( $content ) {
+	$wrap = '<div class="e-content" itemprop="mainContentOfPage">';
+	if ($content!="") {
+		return $wrap . $content . "\n" . '</div>';
+	}
+	return $content;
+}
+
+add_filter( 'the_content', 'independent_publisher_the_content', 1 );
```

Comment:
The problem comes with the design of the_content system in general. The issue involves the desire for markup outside of content to be embedded in content, because plugins use it for display.

Reply:
We can revert to a more traditional methodology. I wish I had a better solution though.